src/wolfgame.py

```diff
@@ -6875,15 +6857,50 @@ def fgame(cli, nick, chan, rest):
             gamemode = gamemode.split()[0]
             gamemode = complete_one_match(gamemode, var.GAME_MODES.keys() - var.DISABLED_GAMEMODES)
             if not gamemode:
-                cli.notice(nick, messages["invalid_mode"].format(rest.split()[0]))
+                wrapper.pm(messages["invalid_mode"].format(message.split()[0]))
                 return
             parts[0] = gamemode
 
-        if cgamemode(cli, "=".join(parts)):
-            cli.msg(chan, messages["fgame_success"].format(nick))
+        if cgamemode("=".join(parts)):
+            channels.Main.send(messages["fgame_success"].format(wrapper.source))
             var.FGAMED = True
     else:
-        cli.notice(nick, fgame.__doc__())
+        wrapper.pm(fgame.__doc__())
+
+@command("frole", flag="d", phases=("join",))
+def frole(var, wrapper, message):
+    """Force a player into a certain role."""
+    pl = get_players()
+    if wrapper.source not in pl and not wrapper.source.is_admin():
+        return
```

Comment:
Why this check? It's a debug mode only command (or will be in the nearish future -- I plan on handling all commands with the "d" flag as debug-mode only and get rid of the debug mode check at the bottom of wolfgame.py) and seems annoying for testers when we are in debug mode because it forces the tester to !join.

Reply:
I'll let @jacob1 fix this command once the PR is merged, so won't touch it further.